spec/helpers/facets_helper_spec.rb

```diff
@@ -0,0 +1,103 @@
+      expect(helper.initial_collapse('pivot_facet', pivot_facet)).to eq('facet-values')
+    end
+
+    it 'returns \'collapse\' for unselected pivot facet' do
+      expect(helper.initial_collapse('pivot_facet', child_item)).to eq('collapse')
+    end
+
+    it 'returns \'collapse show\' for selected pivot facet' do
+      allow(helper).to receive_messages(params: { f: { 'child_item' => ['child value'] } })
+      expect(helper.initial_collapse('pivot_facet', child_item)).to eq('collapse show')
+    end
+  end
+
+  describe 'facet_value_id' do
+    it 'generates an id for pivot facet child items but not for parent items' do
+      expect(helper.facet_value_id(pivot_facet)).to eq ''
+      expect(helper.facet_value_id(child_item)).to eq 'id=child_item-child-value'
+    end
+  end
+
+  describe 'pivot_facet_child_in_params?' do
+    it 'checks if pivot facet child items are not selected' do
+      expect(helper.pivot_facet_child_in_params?('pivot_facet', child_item)).to be false
+      expect(helper.pivot_facet_child_in_params?('pivot_facet', parent_item)).to be false
+      expect(helper.pivot_facet_child_in_params?('pivot_facet', pivot_facet)).to be false
+    end
+
+    it 'checks if pivot facet child items are selected' do
+      allow(helper).to receive_messages(params: { f: { 'child_item' => ['child value'] } })
+      expect(helper.pivot_facet_child_in_params?('pivot_facet', child_item)).to be true
+      expect(helper.pivot_facet_child_in_params?('pivot_facet', parent_item)).to be true
+      expect(helper.pivot_facet_child_in_params?('pivot_facet', pivot_facet)).to be true
+    end
+  end
+
+  describe 'pivot_facet_in_params?' do
+    it 'checks if pivot facet parent item is not selected' do
+      expect(helper.pivot_facet_in_params?('pivot_facet', parent_item)).not_to be true
+    end
+
+    it 'checks if pivot facet parent item is selected' do
+      allow(helper).to receive_messages(params: { f: { 'parent_item' => ['parent value'] } })
+      expect(helper.pivot_facet_in_params?('pivot_facet', parent_item)).to be true
+    end
+  end
+
+  describe 'facet_field_in_params?' do
+    let(:pivot) { ['pivot_a', 'pivot_b'] }
+
+    let(:blacklight_config) do
+      Blacklight::Configuration.new do |config|
+        config.add_facet_field 'basic_field'
+        config.add_facet_field 'other_field'
+        config.add_facet_field 'pivot_facet_field', pivot: ['pivot_a', 'pivot_b']
+      end
+    end
+
+    before do
+      without_partial_double_verification do
+        allow(helper).to receive(:blacklight_config).and_return blacklight_config
+      end
+    end
+
+    it 'checks if the facet field is selected in the user params' do
+      allow(helper).to receive_messages(params: { f: { 'basic-field' => ['x'] } })
+      expect(helper).to be_facet_field_in_params('basic-field')
+      expect(helper.facet_field_in_params?('other-field')).not_to be true
```

Comment:
could you change this to be consistent with the above line:

`      expect(helper).not_to be_facet_field_in_params('other-field')`

this goes for line 113, 118 and 123.